experimental/packages/web-common/src/index.ts

```diff
@@ -18,4 +18,12 @@ export type { SessionProvider } from './types/SessionProvider';
 export {
   createSessionSpanProcessor,
   createSessionLogRecordProcessor,
+  createSessionManager,
+  createDefaultSessionIdGenerator,
 } from './utils';
+export type { Session } from './types/Session';
+export type { SessionIdGenerator } from './types/SessionIdGenerator';
+export type { SessionPublisher } from './types/SessionPublisher';
+export type { SessionObserver } from './types/SessionObserver';
+export type { SessionStore } from './types/SessionStore';
+export { LocalStorageSessionStore } from './LocalStorageSessionStore';
```

Comment:
Looks like we're still exporting a class here. I think this would also be better to use a factory function here and that returns `SessionStore`, to hide any `private` properties from the type in case some are added in the future.

Reply:
This is being exported on purpose because we might add more SessionStore implementations that users could pick from. Is there a better way to handle it if we want to provide multiple implementations in the future?